paddle/fluid/operators/mkldnn/matmul_mkldnn_op.cc

```diff
@@ -145,16 +145,10 @@ class MatMulMKLDNNHandler
     this->AcquireForwardPrimitiveDescriptor(attrs, x_md, y_md, out_md);
   }
   // Constructor for FWD MatMul
-  MatMulMKLDNNHandler(const dnnl::engine engine, const ExecutionContext& ctx,
-                      float scale)
+  MatMulMKLDNNHandler(const dnnl::engine engine, const ExecutionContext& ctx)
       : paddle::platform::MKLDNNHandlerNoCachingT<XT, dnnl::matmul>(
             engine, ctx.GetPlace()) {
-    dnnl::primitive_attr attr;
-    float scale_out = ComputeOutputScale(ctx);
-    if (scale_out != 1.0f) {
-      constexpr unsigned tensor_wide_scale = 0;
-      attr.set_output_scales(tensor_wide_scale, {scale_out});
-    }
+    const dnnl::primitive_attr matmul_attrs = CreatePostOps(ctx);
```

Comment:
You have to change `CreatePostOps` name.
It create matmul_attrs, not postops.

Reply:
I agree, post-ops are part of matmul_attrs and it might be confusing, but that's the pattern. We have same `CreatePostOps` method in which we set attributes and post ops for FC and Conv

Reply:
If the pattern is bad, change it.
This method is private and visible only in this file.
There is no reason to preserve bad names IMHO.